adr/query.py

```diff
@@ -34,7 +34,10 @@ def __init__(self, definitions):
             elif len(definition) >= 2:
                 # Set destination from name
                 definition[1]['dest'] = name
-                self.add_argument(*definition[0], **definition[1])
+                if ("hidden" in definition[1]) and (definition[1]['hidden'] is True):
+                    self.add_argument(*definition[0], help=SUPPRESS)
```

Comment:
I think you'll still need to pass all the other values of `definition[1]` into `add_argument`. I think you need something like this:

    if definition[1].get('hidden'):
        del definition[1]['hidden']
        definition[1]['help'] = SUPPRESS

    self.add_argument(*definition[0]. **definition[1])

Reply:
Yes, this is a bug. The hidden attribute still need default value. I updated it.